specs/merge/beacon-chain.md

```diff
@@ -111,6 +110,25 @@ class ApplicationPayload(Container):
     transactions: List[OpaqueTransaction, MAX_APPLICATION_TRANSACTIONS]
 ```
 
+#### `ApplicationBlockHeader`
+
+The application block header included in a `BeaconState`.
+
+*Note:* Holds application payload data without transaction list.
+
+```python
+class ApplicationBlockHeader(Container):
+    block_hash: Bytes32  # Hash of application block
+    parent_hash: Bytes32
+    coinbase: Bytes20
+    state_root: Bytes32
+    number: uint64
+    gas_limit: uint64
+    gas_used: uint64
+    receipt_root: Bytes32
+    logs_bloom: ByteVector[BYTES_PER_LOGS_BLOOM]
```

Comment:
I suggest we add `transactions_root` so that this merklizes to the same root of `ApplicationPayload`

We can then rename this to `ApplicationPayloadHeader` or rename the payload to `ApplicationBlock`.

(side note, we mgiht be renaming these all to use `Execution` prefix but thats a separate PR